docs/runbook/govnotify-templates.md

```diff
@@ -0,0 +1,90 @@
+# Gov.uk Notify Templates
```

Comment:
Would it be worth linking to the Notify docs? Like https://www.notifications.service.gov.uk/using-notify/guidance maybe?